config/application.rb

```diff
@@ -1,6 +1,19 @@
 require_relative 'boot'
 
-require 'rails/all'
+require 'rails'
+# Pick the frameworks you want:
+require 'action_cable/engine'
+require 'action_controller/railtie'
+require 'action_mailbox/engine'
+require 'action_mailer/railtie'
+# require "action_text/engine"
+require 'action_view/railtie'
+require 'active_job/railtie'
+require 'active_model/railtie'
+require 'active_record/railtie'
+require 'active_storage/engine'
+require 'rails/test_unit/railtie'
+require 'sprockets/railtie'
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
```

Comment:
Below there is a line `config.load_defaults 6.0` it should be `config.load_defaults 6.1` now